src/measure_theory/measure/measure_space.lean

```diff
@@ -643,7 +650,7 @@ if hf : injective f ∧ ∀ s, measurable_set s → measurable_set (f '' s) then
   end
 else 0
 
-lemma comap_apply (f : α → β) (hfi : injective f)
+lemma comap_apply {m0 : measurable_space α} (f : α → β) (hfi : injective f)
```

Comment:
In this one, I don't see how `m0` can be found by Lean. Did you want to put the structure on beta instead?

Reply:
Good catch. It should indeed be on beta. Thanks!